app/controllers/validators_controller.rb

```diff
@@ -68,6 +69,6 @@ def at_33_stake_index(validators, batch, per_page)
   end
 
   def validators_params
-    params.permit(:watchlist, :network, :q, :page, :order, :admin_warning)
+    params.permit(:watchlist, :network, :q, :page, :order, :admin_warning, :jito)
```

Comment:
Let's call this method jito_collaboration as well. To avoid a conflict in the future between this feature, and the one that pulls a list of validators from Jito Pool (currently WIP).